aerich/migrate.py

```diff
@@ -367,38 +367,70 @@ def diff_models(
                 new_data_fields_name = cast(List[str], [i.get("name") for i in new_data_fields])
 
                 # add fields or rename fields
+                rename_fields: Dict[str, str] = {}
```

Comment:
What is the difference between `rename_fields` and `model_rename_fields`? It's a bit confusing to read because there are a lot of variables about renaming but with little information about which part they take in the algorithm: `is_rename`, `rename_fields`, `cls._rename_fields`, `model_rename_fields`.

Reply:
@henadzit Cloud you review again? I removed `model_rename_fields`, and made `rename_fields` more easy to understand.